pkg/forklift-api/webhooks/mutating-webhook/mutators/ovirt-cert-mutator.go

```diff
@@ -28,32 +30,43 @@ func (mutator *OvirtCertMutator) Mutate(ar *admissionv1.AdmissionReview) *admiss
 	secret := &core.Secret{}
 	err := json.Unmarshal(raw, secret)
 	if err != nil {
-		log.Error(err, "mutating webhook error")
-		util.ToAdmissionResponseError(err)
+		log.Error(err, "mutating webhook error, failed to unmarshel certificate")
+		return util.ToAdmissionResponseError(err)
 	}
 
 	if providerType, ok := secret.GetLabels()["createdForProviderType"]; ok && providerType == "ovirt" {
 
 		url, err := url.Parse(string(secret.Data["url"]))
 		if err != nil {
 			log.Error(err, "mutating webhook URL parsing error")
-			util.ToAdmissionResponseError(err)
+			return util.ToAdmissionResponseError(err)
 		}
 
 		certPool := x509.NewCertPool()
-		certPool.AppendCertsFromPEM(secret.Data["cacert"])
+		ok := certPool.AppendCertsFromPEM(secret.Data["cacert"])
+		if !ok {
+			err = liberr.Wrap(errors.New("prasing certificate failure"))
```

Comment:
```suggestion
			err = liberr.Wrap(errors.New("failed to parse certificate"))
```
(there was a typo - 'prasing')